lua/auto-session/init.lua

```diff
@@ -479,11 +480,11 @@ function AutoSession.get_session_files()
   end
 
   local entries = vim.fn.readdir(sessions_dir, function(item)
-    return vim.fn.isdirectory(item) == 0 and not string.find(item, "x.vim$")
+    return Lib.is_session_file(sessions_dir, item)
   end)
 
   return vim.tbl_map(function(entry)
-    return { display_name = AutoSession.format_file_name(entry), path = entry }
+    return { display_name = AutoSession.format_file_name(entry), path = sessions_dir .. "/" .. entry }
```

Comment:
This might be a problem for Windows users?

Reply:
good point. i'll update using a system agnostic separator.